layouts/millennium_r4_sidebar.html

```diff
@@ -15,6 +15,12 @@ <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></
           <li><a href="/millennium/r4/conformance/structure-definition/">StructureDefinition</a></li>
         </ul>
       </li>
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/millennium/r4/care-provision/">Care Provision</a></h3>
+        <ul class="js-guides">
+          <li><a href="/millennium/r4/care-provision/nutrition-order/">NutritionOrder</a></li>
+        </ul>
+      </li>
```

Comment:
Should this be alphabetical order?

Reply:
It is ordered by 'Care Provision'. May be i am missing something here?

Reply:
I was more so speaking of Care-Provision coming before Conformance

Reply:
I think he might be suggesting that "Care Provision" should come before "Conformance"? But "Overview" is on the top and "Diagnostic" is next, so I don't see any order...